integrations/airflow/README.md

```diff
@@ -1,154 +1,9 @@
-# marquez-airflow
+# DEPRECATED
```

Comment:
We will still be publishing the `marquez-airflow` lib, so I wouldn't mark the module as deprecated. All the other changes to the `README.md` look good.

Reply:
Changed.